### ISSUE TITLE
[req] RUDUB

### ISSUE THREAD

**Issue by SuberSeb** [Jackett/Jackett#5673]
Baibako.tv is a Russian semi-private tracker. It has a lot of TV shows. http://baibako.tv
Became RUDUB

**Comment by garfield69** [comment]
try out jackett pre-release 0.11.512

**Comment by SuberSeb** [comment]
Found 0 releases from BaibaKo for: Rook S01E01
Search from Sonarr doesn't work.

**Comment by SuberSeb** [comment]
Seems like that baibako needs a pre-process step to convert SxxExx into Сезон xx Серия xx so indexer can find it.

**Comment by garfield69** [comment]
logon to the site directly and test searching as you specified.
report back what you searched for, and the url generated by the browser for the successful search.

**Comment by SuberSeb** [comment]
I searched this: "city on a hill /s01e01-06" for episodes 01-06 (it a feature of russian trackers, all episodes in one torrent, it's normal). Slash in query is necessary, without it query returns none.
Here is the url: http://baibako.tv/browse.php?search=city+on+a+hill+%2Fs01e01-06&incldead=0&cat=0&videoformat=0

**Comment by garfield69** [comment]
could be a problem. if sonarr looks for `city on a hill s01e06`
are we going to have to alter the query to `city on a hill /s01e01-06` to get a hit?

**Comment by SuberSeb** [comment]
Yes, I think. By the way other trackers such as Rutracker or NNM Club works in similar way, there are no separate torrents for each episodes, only one for the all episodes in season.

**Comment by garfield69** [comment]
yes, but with rutracker we grab the season, drop leading zeroes, and search for `city on a hill Сезон 1`
and then hack the result title back into something like `city on a hill s1e1-6` for sonarr to digest.
i cant find any preprocessing in nnmclub, I don't think its set up for sonarr although there is the same title hacking of the results.

But `city on a hill Сезон 1` does not work on baibako. 
it looks like it only searches for exact matching, `city hill` and any form of mask using % * ? does not work.
the sites search engine sucks.

so i'll have to hack something up from scratch based on the required /s01 key.
see if I can make something work.

**Comment by garfield69** [comment]
added keywords-filter to convert the `SxxExx` from the query into `/sxx`
seems to work for me on my Sonarr test.
try out jackett pre-release 0.11.514

**Comment by SuberSeb** [comment]
It works, thank you. But I have error "Russian is wanted but found English".

**Comment by garfield69** [comment]
have you go Russian as the filter langiage for Sonarr?
i could add RUS to the title. I think that might satisfy the filter.

**Comment by SuberSeb** [comment]
Yes, I have russian filter in Sonarr. I think RUS in title solve the problem.

**Comment by garfield69** [comment]
try out jackett pre-release 0.11.517

**Comment by SuberSeb** [comment]
Now all works well, thank you.

**Comment by dimqua** [comment]
https://github.com/Jackett/Jackett/commit/6e8726c197cc2de30420166dc676b9490bf2b818

@garfield69 isn't it still semi-private?

**Comment by SuberSeb** [comment]
Yes, it still semi-private.

**Comment by garfield69** [comment]
When I tested this 6 days ago I could not find the registration page hence the **private** flag.
but I see that the registration page is back so I will revert.

**Comment by ilike2burnthing** [comment]
I'm getting ERR_CONNECTION_TIMED_OUT for https://baibako.tv/ but not sure if it's just me. Failing in Jackett as well with a proxy/VPN.

https://www.uptrends.com/tools/uptime is showing it as up for most but then several come back with TCP Connection Failed.

Anyone else having issues?

**Comment by SuberSeb** [comment]
> I'm getting ERR_CONNECTION_TIMED_OUT for https://baibako.tv/ but not sure if it's just me. Failing in Jackett as well with a proxy/VPN.
> 
> https://www.uptrends.com/tools/uptime is showing it as up for most but then several come back with TCP Connection Failed.
> 
> Anyone else having issues?

Test successful for me now. Is it available from your browser?

**Comment by ilike2burnthing** [comment]
Nope :/ ERR_CONNECTION_TIMED_OUT

**Comment by SuberSeb** [comment]
Same error here... Not sure why this happening.

**Comment by garfield69** [comment]
@SuberSeb  thanks for the info

**Comment by garfield69** [comment]
Jackett 0.16.1883

**Comment by garfield69** [comment]
http://bko.baibako.tv/ returning ERR_EMPTY_RESPONSE

**Comment by SuberSeb** [comment]
For me site is not working. They have problems with the site from time to time. Let's wait some time.

**Comment by ilike2burnthing** [comment]
ERR_NAME_NOT_RESOLVED

**Comment by SuberSeb** [comment]
They said that they have a DDOS attack on the site, so it's not working now.

**Comment by SuberSeb** [comment]
They have new site - http://rudub.tv/. Do we need new indexer?

**Comment by ilike2burnthing** [comment]
Is this a permanent change or just while they get baibako.tv back online? Is baibako.tv dead?

Where are you getting these updates?

**Comment by SuberSeb** [comment]
> Is this a permanent change or just while they get baibako.tv back online? Is baibako.tv dead?

Baibako.tv seems to be dead, so I assume that this is permanent change. I doesn't see any reasons for Baibako.tv back online.

> Where are you getting these updates?

In various communities where people want to watch TV shows :)

**Comment by garfield69** [comment]
@ilike2burnthing  you forgot to move the old domain into legacy and add the new domain as the default link ;-)

**Comment by ilike2burnthing** [comment]
Got distracted correcting the spelling and capitalisation of `ex-BaibaKoTV` that I completely forgot the main reason I was editing the file in the first place ;P

**Comment by garfield69** [comment]
v0.20.703

**Comment by garfield69** [comment]
https://apr11.rudub.vip/login.php > 500 Internal Server Error nginx/1.18.0
https://rudub.vip/ > HTTP ERROR 520

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://jun05.rudub.run/ > 500 Internal Server Error nginx/1.18.0
https://jun06.rudub.run/ > 500 Internal Server Error nginx/1.18.0

**Comment by ilike2burnthing** [comment]
https://jun06.rudub.run/ now working

**Comment by garfield69** [comment]
https://jun20.rudub.icu/ > Web server is returning an unknown error Error code 520
back to https://jun19.rudub.icu/ > OK

**Comment by garfield69** [comment]
https://jun20.rudub.icu/ online

**Comment by garfield69** [comment]
https://dec08.rudub.world/ > DNS Lookup Error

**Comment by garfield69** [comment]
found a bunch of alternate domains
  - https://dec08.rudub.bet/
  - https://dec08.rudub.buzz/
  - https://dec08.rudub.homes/
  - https://dec08.rudub.mom/
  - https://dec08.rudub.online/
  - https://dec08.rudub.shop/
  - https://dec08.rudub.top/
  - https://dec08.rudub.xyz/
  
I've switched to bet

@mynameisbogdan  do you think you could update https://github.com/Jackett/Jackett/blob/master/.github/workflows/definitions-update.yml to update multiple tld?
current valid tld are bet, buzz, homes, mom, online, shop, top and xyz

**Comment by mynameisbogdan** [comment]
Should be doable, but are we going to add 30 old legacy links per tld then? 

Unless they're blocked, one domain should be enough for this indexer IMO. I'm surprised you still want to support it...

**Comment by garfield69** [comment]
Agreed.
I'll just add the TLDs as a comment in the indexer so if/when the first one breaks and the usual redirect does not provide an alternate then I will just update the indexer to switch to one of the other TLDs.

**Comment by mynameisbogdan** [comment]
Worst case scenario, we can add a secondary if the need occurs or if you desire this.

**Comment by garfield69** [comment]
rudub have changed domain structure. I have disabled the definitions-update action for now.
We will know tomorrow if it is still required, with modifications.

**Comment by ilike2burnthing** [comment]
Remove [definitions-update.yml](https://github.com/Jackett/Jackett/blob/master/.github/workflows/definitions-update.yml) ?
